packages/puppeteer-core/src/bidi/core/BrowsingContext.ts

```diff
@@ -608,10 +610,22 @@ export class BrowsingContext extends EventEmitter<{
   async setScreenOrientationOverride(
     screenOrientation: Bidi.Emulation.ScreenOrientation | null,
   ): Promise<void> {
-    await this.#session.send('emulation.setScreenOrientationOverride', {
-      screenOrientation,
-      contexts: [this.id],
-    });
+    try {
+      await this.#session.send('emulation.setScreenOrientationOverride', {
+        screenOrientation,
+        contexts: [this.id],
+      });
+    } catch (err) {
+      // Tolerate ProtocolError for
```

Comment:
Please add "TODO: remove on ..." with a specific date when FF 144 is going stable.